### PR TITLE
ZK-4569: MVVM support calling commands in other view model

### PR DESCRIPTION
**Pull request by DevChu**

### CODE REVIEW

File: zkbind/src/org/zkoss/bind/BindComposer.java [hunk 1]
```diff
@@ -442,6 +442,15 @@ private AnnotateBinder initBinder(BindEvaluatorX evalx, Component comp) {
 		comp.setAttribute(bname, binder);
 		comp.setAttribute(BINDER_ID, bname);
 
+		//ZK-4569
+		Desktop desktop = comp.getDesktop();
+		Map<String, Object> relationMap = (Map<String, Object>) desktop.getAttribute(BinderCtrl.VIEWMODELID_BINDER_MAP_KEY);
```
Comment (scribetw) [inline zkbind/src/org/zkoss/bind/BindComposer.java:447]: Binder

File: zktest/src/archive/test2/F91-ZK-4569.zul [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		 height="500px">
+		<div hflex="1" vflex="1">
+			<label multiline="true"><![CDATA[
+				Click buttons, and see
+					Clicked
+					Clicked
+					Clickedb
+					ClickedG
+			]]></label>
+			<button label="click1" onClick="@command('doClick')" />
```
Comment (scribetw) [inline zktest/src/archive/test2/F91-ZK-4569.zul:24]: click 1,2,3...

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**